backend-kotlin/src/main/kotlin/com/example/backendkotlin/infrastructure/db/SchemaInitialize.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.backendkotlin.infrastructure.db
+
+import com.example.backendkotlin.infrastructure.db.table.RUserVillageTable
+import com.example.backendkotlin.infrastructure.db.table.UserTable
+import com.example.backendkotlin.infrastructure.db.table.VillageTable
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.springframework.boot.ApplicationArguments
+import org.springframework.boot.ApplicationRunner
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+
+@Component
+@Transactional
+class SchemaInitialize : ApplicationRunner {
+    override fun run(args: ApplicationArguments?) {
```

Comment:
ここ nullable 消してもよいかも
```suggestion
    override fun run(args: ApplicationArguments) {
```

Reply:
修正
d0c6de5